### DISCUSSION TITLE
Representing selection process in PRISMA flow diagram

### DISCUSSION THREAD

**Discussion by DBergman1997** [asreview/asreview#1202] [Q&A] [answered]
My co-reviewer and I struggle a bit with representing the selection process of our review in the PRISMA flow diagram in our systematic review article. We would like to ask if someone has any examples of such diagrams from review articles that used ASReview

**Comment by boer0107-zz** [comment, marked as answer]
We maintain a [list of systematic reviews in which ASReview is used](https://www.zotero.org/groups/4597652/asreview_public/collections/IWU8ATMK/tags/peer_reviewed_article,systematic_review/collection) 
I have seen some nice examples, but can't recall which ones. Let us know which one appealed to you the most.
